CMakeLists.txt

```diff
@@ -75,6 +76,20 @@ if (USE_OPENMP)
   endif ()
 endif ()
 
+set (LIBCMAES_DEPS "")
+set (TBB_DEFS "")
+
+if (USE_TBB)
+  find_package (TBB)
+  if (TBB_FOUND)
+    set (TBB_DEFS "-DHAVE_TBB")
+    set (LIBCMAES_DEPS "${TBB_LIBRARIES}")
```

Comment:
It's useless to define LIBCMAES_DEPS as it's conditionally linked anyways, so you can use TBB_LIBRARIES directly.

Reply:
It's useless to fight that much for things that are harmless. Will change them nevertheless..